content_scripts/inject_script.coffee

```diff
@@ -0,0 +1,16 @@
+injectScripts = ["pages/addEventListener_hook.js"]
+
+fetchFileContents = (extensionFileName) ->
+  req = new XMLHttpRequest()
+  req.open("GET", chrome.runtime.getURL(extensionFileName), false) # false => synchronous
+  req.send()
+  req.responseText
```

Comment:
The plan was for it to be temporary, so this will disappear shortly when I do what I've said [here](https://github.com/philc/vimium/pull/1859#issuecomment-149070551)